File: App2/src/app/_core/_model/account-group.ts
export interface AccountGroup {
  id: number;
  zoneID: number;
  buildingID: number;
  groupNO: string;
  groupName: string;
  status: boolean | null;
  createBy: number;
  updateBy: number | null;
  deleteBy: number | null;
  createDate: string;
  updateDate: string | null;
  deleteDate: string | null;
}

File: PigFarm/Models/AccountGroup.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace PigFarm.Models
{
    public partial class AccountGroup
    {
        public int Id { get; set; }
        public int ZoneId { get; set; }
        public int BuildingId { get; set; }
        public string GroupNo { get; set; }
        public string GroupName { get; set; }
        public bool? Status { get; set; }
        public int CreateBy { get; set; }
        public int? UpdateBy { get; set; }
        public int? DeleteBy { get; set; }
        public DateTime CreateDate { get; set; }
        public DateTime? UpdateDate { get; set; }
        public DateTime? DeleteDate { get; set; }
    }
}
